project_task_description_template/models/project_task.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (C) 2023 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+from odoo import api, fields, models
+
+
+class ProjectTask(models.Model):
+    _inherit = "project.task"
+
+    task_template_id = fields.Many2one(comodel_name="project.task.template")
+    template_visible = fields.Boolean(compute="_compute_template_visible")
+
+    @api.depends("project_id", "stage_id")
```

Comment:
If you use `depends` for the compute field then field must be `store=True`.

Reply:
okay, I'll fix it

Reply:
> If you use `depends` for the compute field then field must be `store=True`.

That's not necessarily true. Even if you don't store the result, you may want it recomputed as certain fields change. It's not a mistake to have an `api.depends` with a field that is not stored IMO.

Reply:
is this comment still relevant or can it already be closed?

Reply:
if the field is working as expected, I think it can be closed